py-polars/tests/unit/expr/test_exprs.py

```diff
@@ -747,3 +747,16 @@ def test_slice_rejects_non_integral() -> None:
 
     with pytest.raises(pl.exceptions.InvalidOperationError):
         df.select(pl.col("a").slice(pl.lit("1"), None)).collect()
+
+
+def test_slice() -> None:
```

Comment:
Can you test what you want fixed? E.g. ensure that we raise on the invalid input.